rotkehlchen/db/filtering.py

```diff
@@ -470,6 +472,22 @@ def make(
         if location is not None:
             filter_query.location_filter = DBLocationFilter(and_op=True, location=location)
             filters.append(filter_query.location_filter)
+        if multiple_base_assets is not None:
```

Comment:
Why go through this and not make the base_asset and the quote asset a multifield? This reads like a terrible hack.You are introducing two new arguments in the filter that already exist, but just call them, multi?

Reply:
Okey, will change the approach here. Should be the same for the others? I understand so

Reply:
What I mean is to make the base/quote asset have the ability to be multiple assets and not have a separate argument in the case they may be multiple.